### ISSUE TITLE
namespace options ignored for helm deployments

### ISSUE THREAD

**Issue by shaunc** [devspace-cloud/devspace#1015]
**What happened?**  
With configuration:
```
version: v1beta7
deployments:
- name: foo
  namespace: foo
  helm:
    componentChart: true
    values:
      containers:
      - image: busybox
```
When I
```
$ kubectl create namespace foo
$ devspace deploy
```
Then the deployment ends up in the current namespace, not in `foo`:
```
$ kubectl get -n foo all
No resources found in foo namespace.
$ kubectl get deployment foo
NAME      READY   UP-TO-DATE   AVAILABLE   AGE
foo   0/1     1            0           4m11s
```

**What did you expect to happen instead?**  

Helm would deploy to namespace `foo` rather than the current namespace.

**How can we reproduce the bug?** (as minimally and precisely as possible)  

Should be sufficient

**Local Environment:**  
- DevSpace Version: [use `devspace --version`]
```
% devspace --version
devspace version 4.6.2
```
- Operating System: mac
- Deployment method: helm 

**Kubernetes Cluster:**  
- Cloud Provider: aws
- Kubernetes Version: [use `kubectl version`]
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T23:41:55Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.9-eks-502bfb", GitCommit:"502bfb383169b124d87848f89e17a04b9fc1f6f0", GitTreeState:"clean", BuildDate:"2020-02-07T01:31:02Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}

**Anything else we need to know?**  

It seems to me that this use to be working....

/kind bug

**Comment by FabianKramm** [comment]
@shaunc thanks for reporting! I'll fix this ASAP.

**Comment by FabianKramm** [comment]
Should be fixed with v4.7.0
